src/mongo/Driver.class.php

```diff
@@ -31,19 +31,6 @@ class Driver extends DriverBase implements \Tripod\IDriver
      */
     private $search_indexer = null;
 
-
-    /**
-     * @var array The original read preference gets stored here
-     *            when changing for a write.
-     */
-    private $originalCollectionReadPreference = array();
```

Comment:
I don't actually see any indication that they were referenced anywhere.  It looks like it might have been intended to be what the same properties on Updates.class.php became.